src/index.ts

```diff
@@ -90,6 +92,58 @@ async function setDebugSession(
   app.commands.notifyCommandChanged();
 }
 
+function updateToolbar(
+  widget: NotebookPanel | ConsolePanel | FileEditor,
+  debug: IDebugger,
+  app: JupyterFrontEnd
```

Comment:
Instead of passing the whole application, I would suggest just passing the `CommandRegistry` that executes the command you need, because you're not using any other app functionality.